src/zzz_od/api/routers/battle_assistant.py

```
@@ -0,0 +1,2509 @@
+    import requests
+    data = {
+        "use_gpu": True,
+        "screenshot_interval": 0.03,
+        "gamepad_type": "xbox"
+    }
+    response = requests.put("http://localhost:8000/api/v1/battle-assistant/settings", json=data)
+    print(response.json()["message"])
+    ```
+    """
+    try:
+        # 更新GPU使用设置（如果提供）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)